spec/unit/pops/loaders/loaders_spec.rb

```diff
@@ -52,6 +52,10 @@
 
   # Loaders caches the puppet_system_loader, must reset between tests
 
+  before :each do
+    allow(File).to receive(:read).and_call_original
```

Comment:
Is this just undoing stubs?

Reply:
The test stubs/expects calls to `File.read` with specific arguments. However, in ruby 2.6, ruby's `Pathname#read` now calls `File.read` too. So in order for Pathname to still work, we have to allow other `File.read` calls to pass through, as though the method was not stubbed. However, I should probably make this more specific to the files that the test is operating on, as opposed to all files. Let me update that.

Reply:
On second thought, there are lots of files loaded during normal compilation, and trying to special case them will be fragile, e.g. `"/Users/josh/work/puppet/lib/puppet/functions/include.rb"`. The test is asserting that the functions return the correct output, which ensures that the correct function was loaded. So I'm going to keep the generic allow file read to call original.